eleventy.config.js

```diff
@@ -130,7 +129,7 @@ module.exports = function (config) {
             ready: function (err, bs) {
 
                 bs.addMiddleware("*", (req, res) => {
-                    const content_404 = fs.readFileSync("dist/404.html");
+                    const content_404 = fs.readFileSync("_site/404.html");
```

Comment:
The whole BrowserSync config block starting on line 126 should be removed, as BrowserSync is no longer used: https://www.11ty.dev/docs/server-browsersync/